content/applications/sales/sales/invoicing/proforma.rst

```diff
@@ -1,34 +1,54 @@
-========================
-Send a pro-forma invoice
-========================
+==================
+Pro-forma invoices
+==================
+
+A *Pro-Forma Invoice* is an abridged or estimated invoice sent in advance of a delivery of goods.
+It notes the kind and quantity of goods, their value, and other important information, such as
+weight and transportation charges.
 
-A pro-forma invoice is an abridged or estimated invoice in advance of a
-delivery of goods. It notes the kind and quantity of goods, their value,
-and other important information such as weight and transportation
-charges. Pro-forma invoices are commonly used as preliminary invoices
-with a quotation, or for customs purposes in importation. They differ
-from a normal invoice in not being a demand or request for payment.
+Pro-forma invoices are commonly used as preliminary invoices with a quotation. They are also used
+for customs purposes during importation. They differ from a normal invoice in that they are not a
+demand (or request) for payment.
```

Comment:
This makes me wonder what the difference is between a pro-forma invoice and a quotation. It might be good to add a brief explanation.

Reply:
I understand, but I do think the explanation can be found on lines 5-7. And, considering this doc is solely focused on pro-forma invoices, I believe the reader/audience is already aware of what a common quotation is...but again, I completely get where you're coming from. But, in this instance, I think, for the sake of staying as concise as possible, I'm going to leave this portion as-is. But, I genuinely do appreciate the suggestion. Thanks! 👍